/home/kballou/Dev/NBody/nbody/kernels.py:54: UserWarning: nvcc said it demoted types in source code it compiled--this is likely not what you want.
  add_mod = SourceModule(add_template.render(kwargs))
/home/kballou/Dev/NBody/nbody/kernels.py:54: UserWarning: The CUDA compiler suceeded, but said the following:
ptxas /tmp/tmpxft_00000a29_00000000-5_kernel.ptx, line 90; warning : Double is not supported. Demoting to float

  add_mod = SourceModule(add_template.render(kwargs))
/home/kballou/Dev/NBody/nbody/kernels.py:54: UserWarning: nvcc said it demoted types in source code it compiled--this is likely not what you want.
  add_mod = SourceModule(add_template.render(kwargs))
/home/kballou/Dev/NBody/nbody/kernels.py:54: UserWarning: The CUDA compiler suceeded, but said the following:
ptxas /tmp/tmpxft_00000811_00000000-5_kernel.ptx, line 90; warning : Double is not supported. Demoting to float

  add_mod = SourceModule(add_template.render(kwargs))
/home/kballou/Dev/NBody/nbody/kernels.py:54: UserWarning: nvcc said it demoted types in source code it compiled--this is likely not what you want.
  add_mod = SourceModule(add_template.render(kwargs))
/home/kballou/Dev/NBody/nbody/kernels.py:54: UserWarning: The CUDA compiler suceeded, but said the following:
ptxas /tmp/tmpxft_000009ec_00000000-5_kernel.ptx, line 90; warning : Double is not supported. Demoting to float

  add_mod = SourceModule(add_template.render(kwargs))
/home/kballou/Dev/NBody/nbody/kernels.py:54: UserWarning: nvcc said it demoted types in source code it compiled--this is likely not what you want.
  add_mod = SourceModule(add_template.render(kwargs))
/home/kballou/Dev/NBody/nbody/kernels.py:54: UserWarning: The CUDA compiler suceeded, but said the following:
ptxas /tmp/tmpxft_00000816_00000000-5_kernel.ptx, line 90; warning : Double is not supported. Demoting to float

  add_mod = SourceModule(add_template.render(kwargs))
/home/kballou/Dev/NBody/nbody/kernels.py:54: UserWarning: nvcc said it demoted types in source code it compiled--this is likely not what you want.
  add_mod = SourceModule(add_template.render(kwargs))
/home/kballou/Dev/NBody/nbody/kernels.py:54: UserWarning: The CUDA compiler suceeded, but said the following:
ptxas /tmp/tmpxft_00000784_00000000-5_kernel.ptx, line 90; warning : Double is not supported. Demoting to float

  add_mod = SourceModule(add_template.render(kwargs))
Traceback (most recent call last):
  File "run.py", line 17, in <module>
    slave.main(RANK)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 124, in main
    compute(particles)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 28, in compute
    R)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 118, in compute_grid
    compute_mesh(mesh_64, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 115, in compute_mesh
    kernel_mesh(grids, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 75, in kernel_mesh
    cuda.In(grids_m), block=block)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 355, in function_call
Traceback (most recent call last):
  File "run.py", line 17, in <module>
    slave.main(RANK)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 124, in main
    compute(particles)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 28, in compute
    R)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 118, in compute_grid
    compute_mesh(mesh_64, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 115, in compute_mesh
    kernel_mesh(grids, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 75, in kernel_mesh
    cuda.In(grids_m), block=block)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 355, in function_call
    handlers, arg_buf = _build_arg_buf(args)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 125, in _build_arg_buf
    arg_data.append(int(arg.get_device_alloc()))
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 59, in get_device_alloc
    self.dev_alloc = mem_alloc_like(self.array)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 608, in mem_alloc_like
    handlers, arg_buf = _build_arg_buf(args)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 125, in _build_arg_buf
    arg_data.append(int(arg.get_device_alloc()))
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 59, in get_device_alloc
    self.dev_alloc = mem_alloc_like(self.array)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 608, in mem_alloc_like
    return mem_alloc(ary.nbytes)
pycuda._driver.LogicError: cuMemAlloc failed: invalid value
    return mem_alloc(ary.nbytes)
pycuda._driver.LogicError: cuMemAlloc failed: invalid value
Traceback (most recent call last):
  File "run.py", line 17, in <module>
    slave.main(RANK)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 124, in main
    compute(particles)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 28, in compute
    R)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 118, in compute_grid
    compute_mesh(mesh_64, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 115, in compute_mesh
    kernel_mesh(grids, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 75, in kernel_mesh
    cuda.In(grids_m), block=block)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 355, in function_call
Traceback (most recent call last):
  File "run.py", line 17, in <module>
    slave.main(RANK)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 124, in main
    compute(particles)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 28, in compute
    R)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 118, in compute_grid
    compute_mesh(mesh_64, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 115, in compute_mesh
    kernel_mesh(grids, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 75, in kernel_mesh
    cuda.In(grids_m), block=block)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 355, in function_call
    handlers, arg_buf = _build_arg_buf(args)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 125, in _build_arg_buf
    arg_data.append(int(arg.get_device_alloc()))
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 59, in get_device_alloc
    self.dev_alloc = mem_alloc_like(self.array)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 608, in mem_alloc_like
    return mem_alloc(ary.nbytes)
pycuda._driver.LogicError: cuMemAlloc failed: invalid value
Traceback (most recent call last):
  File "run.py", line 17, in <module>
    slave.main(RANK)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 124, in main
    compute(particles)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 28, in compute
    R)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 118, in compute_grid
    compute_mesh(mesh_64, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 115, in compute_mesh
    kernel_mesh(grids, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 75, in kernel_mesh
    cuda.In(grids_m), block=block)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 355, in function_call
Traceback (most recent call last):
  File "run.py", line 17, in <module>
    slave.main(RANK)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 124, in main
    compute(particles)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 28, in compute
    R)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 118, in compute_grid
    compute_mesh(mesh_64, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 115, in compute_mesh
    kernel_mesh(grids, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 75, in kernel_mesh
    cuda.In(grids_m), block=block)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 355, in function_call
Traceback (most recent call last):
  File "run.py", line 17, in <module>
    slave.main(RANK)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 124, in main
    compute(particles)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 28, in compute
    R)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 118, in compute_grid
    compute_mesh(mesh_64, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 115, in compute_mesh
    kernel_mesh(grids, grid_spread)
  File "/home/kballou/Dev/NBody/nbody/slave.py", line 75, in kernel_mesh
    cuda.In(grids_m), block=block)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 355, in function_call
    handlers, arg_buf = _build_arg_buf(args)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 125, in _build_arg_buf
    arg_data.append(int(arg.get_device_alloc()))
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 59, in get_device_alloc
    self.dev_alloc = mem_alloc_like(self.array)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 608, in mem_alloc_like
    return mem_alloc(ary.nbytes)
pycuda._driver.LogicError: cuMemAlloc failed: invalid value
    handlers, arg_buf = _build_arg_buf(args)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 125, in _build_arg_buf
    arg_data.append(int(arg.get_device_alloc()))
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 59, in get_device_alloc
    self.dev_alloc = mem_alloc_like(self.array)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 608, in mem_alloc_like
    return mem_alloc(ary.nbytes)
    handlers, arg_buf = _build_arg_buf(args)
pycuda._driver.LogicError: cuMemAlloc failed: invalid value  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 125, in _build_arg_buf

    arg_data.append(int(arg.get_device_alloc()))
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 59, in get_device_alloc
    self.dev_alloc = mem_alloc_like(self.array)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 608, in mem_alloc_like
    return mem_alloc(ary.nbytes)
pycuda._driver.LogicError: cuMemAlloc failed: invalid value
    handlers, arg_buf = _build_arg_buf(args)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 125, in _build_arg_buf
    arg_data.append(int(arg.get_device_alloc()))
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 59, in get_device_alloc
    self.dev_alloc = mem_alloc_like(self.array)
  File "/usr/lib/python2.7/site-packages/pycuda/driver.py", line 608, in mem_alloc_like
    return mem_alloc(ary.nbytes)
pycuda._driver.LogicError: cuMemAlloc failed: invalid value
/home/kballou/Dev/NBody/nbody/kernels.py:54: UserWarning: The CUDA compiler suceeded, but said the following:
ptxas /tmp/tmpxft_00000a45_00000000-5_kernel.ptx, line 85; warning : Double is not supported. Demoting to float

  add_mod = SourceModule(add_template.render(kwargs))
/home/kballou/Dev/NBody/nbody/kernels.py:54: UserWarning: The CUDA compiler suceeded, but said the following:
ptxas /tmp/tmpxft_0000082d_00000000-5_kernel.ptx, line 85; warning : Double is not supported. Demoting to float

  add_mod = SourceModule(add_template.render(kwargs))
/home/kballou/Dev/NBody/nbody/kernels.py:54: UserWarning: The CUDA compiler suceeded, but said the following:
ptxas /tmp/tmpxft_00000832_00000000-5_kernel.ptx, line 85; warning : Double is not supported. Demoting to float

  add_mod = SourceModule(add_template.render(kwargs))
mpirun: killing job...

Traceback (most recent call last):
  File "/home/kballou/.local/bin/mpi_run.py", line 17, in <module>
    err = subprocess.call(mpi_cmd)
  File "/usr/lib/python3.3/subprocess.py", line 522, in call
    return p.wait(timeout=timeout)
  File "/usr/lib/python3.3/subprocess.py", line 1528, in wait
    (pid, sts) = self._try_wait(0)
  File "/usr/lib/python3.3/subprocess.py", line 1485, in _try_wait
    (pid, sts) = _eintr_retry_call(os.waitpid, self.pid, wait_flags)
  File "/usr/lib/python3.3/subprocess.py", line 476, in _eintr_retry_call
    return func(*args)
KeyboardInterrupt
Command exited with non-zero status 1
0.13user 0.02system 0:36.40elapsed 0%CPU (0avgtext+0avgdata 6840maxresident)k
0inputs+24outputs (0major+18069minor)pagefaults 0swaps
